### DISCUSSION TITLE
Chance for agressive dolphins to spawn

### DISCUSSION THREAD

**Discussion by RhythmicSys** [PurpurMC/Purpur#622] [__Completed__] [answered]
Noticed a little while ago that dolphins are, apparently, neutral mobs and will attack you if you attack them.
Now I wanna see attack dolphins that are just agressive spawn :)

**Comment by BillyGalbreath** [comment, marked as answer]
Added in build 1361

https://github.com/pl3xgaming/Purpur/commit/42969ccbe206a65d062a3bfe2bba34a1167b3ad9
